package.json

```diff
@@ -52,7 +52,6 @@
     "color": "^2.0.1",
     "d3": "^4.13.0",
     "debug": "^3.1.0",
-    "electron-is-running-in-asar": "^1.0.0",
```

Comment:
Should the shrinkwrap file have updated too? :man_shrugging:

Reply:
no

Reply:
It is also required by resin-corvus.